index.html

```diff
@@ -128,7 +128,9 @@ <h1>I'm Majek
     </div>
 
     <div class="about-myself">
-      <h1>About <br> Myself</h1>
+      <div class="heading-3">
+        <hx>About <br> Myself</hx>
```

Comment:
the label Hx does not exist, the "x" symbolizes the number of importance of the label for example H1, H2, etc.